crates/eww/src/config/system_stats.rs

```diff
@@ -229,3 +231,17 @@ pub fn net() -> String {
 pub fn get_time() -> String {
     chrono::offset::Utc::now().timestamp().to_string()
 }
+
+pub fn get_ipv4() -> String {
+    let ifas = list_afinet_netifas().unwrap();
```

Comment:
please don't crash the application.
make use of the error [the function](https://docs.rs/local-ip-address/0.6.1/local_ip_address/linux/fn.list_afinet_netifas.html) returns.